### ISSUE TITLE
Add versioning and release tags

### ISSUE THREAD

**Issue by gauthamchandra** [Janhouse/tespeed#35]
The code currently has no versioning and it would be nice to have versioning on this.

Once versioning and release tags are setup, I can open a PR for a homebrew formula for this utility so that new users can just do:

`brew install tespeed`

According to homebrew docs:

> Formulae in the core repository must have a stable version tagged by the upstream project. Tarballs are preferred to git checkouts, and tarballs should include the version in the filename whenever possible.

[Full details on Homebrew Versioning](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#stable-versions)

You _could_ also just have a bleeding edge formula that might break when users run `brew update` but its not optimal.

**Comment by thanegill** [comment]
👍

**Comment by Janhouse** [comment]
Good idea, I added v1.0 tag.

**Comment by gauthamchandra** [comment]
I decided to go via `pip` instead of via `homebrew` as its a python module. 

I have gotten almost everything setup but I am just a bit stuck on this annoying issue of it not resolving the custom SocksiPy lib you have in your repo.

Seems pip doesn't like the fact that it's a submodule. 

I am going to see if I can hack around it with requirements files.

**Comment by Janhouse** [comment]
Yes, seems like Github didn't want to include it in the release automatically. :sob:

**Comment by gauthamchandra** [comment]
Just need a code review and I am good to go. :+1:
